src/aurelia-authentication.js

```diff
@@ -35,8 +35,15 @@ function configure(aurelia, config) {
     });
   }
 
+  let client;
+
   // Let's see if there's a configured client.
-  let client = clientConfig.getEndpoint(baseConfig.current.endpoint);
+  if (baseConfig.current.endpoint) {
```

Comment:
I missed the fact that this defaults to null, my bad.

I still prefer (strongly) doing stricter checks in these cases. So, endpoint !== null. If you don't want to, I still :+1:. But I had to say it :)